lib/libtlspool_posix.c

```diff
@@ -0,0 +1,155 @@
+#include "whoami.h"
```

Comment:
deze enorme switch is een reden waarom een fork me echt beter lijkt, naast alle andere argumenten

Reply:
ik doe nu in lib/libtlspool.c

#ifdef WINDOWS_PORT
#include "libtlspool_windows.c"
#else
#include "libtlspool_posix.c"
#endif

in mijn fork zit dan wel libtlspool_windows.c